hydroflow/src/scheduled/graph.rs

```diff
@@ -284,6 +287,7 @@ impl Hydroflow {
 
                 self.context.current_stratum = 0;
                 self.context.current_tick += 1;
+                self.context.current_tick_start = Instant::now();
```

Comment:
seems like this won't be set during the first tick?

Reply:
I changed how it works to initially set the value when run_tick/run_tick_async/run_available/run_available_async is called. We should probably make run_stratum private because it is really hard (maybe impossible?) for a user to use it correctly.